src/main/scala/com/mwz/sonar/scala/scapegoat/ScapegoatSensor.scala

```diff
@@ -165,8 +169,12 @@ final class ScapegoatSensor(scapegoatReportParser: ScapegoatReportParserAPI) ext
 
 private[scapegoat] object ScapegoatSensor {
   final val SensorName = "Scapegoat Sensor"
+  final val ScapegoatDisablePropertyKey = "sonar.scala.scapegoat.disable"
   final val ScapegoatReportPathPropertyKey = "sonar.scala.scapegoat.reportPath"
 
+  def shouldDisableSensor(settings: Configuration): Boolean =
+    settings.get(ScapegoatDisablePropertyKey).toOption.exists(s => s.toLowerCase == "true")
```

Comment:
What about `flatMap(_.toBooleanOpt).getOrElse(false)` ?

Reply:
I inverted the logic in 2f12199, could you check if you're happy with that now?